ethereumj-core/src/main/java/org/ethereum/net/rlpx/EncryptionHandshake.java

```diff
@@ -360,11 +360,11 @@ public ECPoint getRemotePublicKey() {
             return token;
         }
 
-        public SHA3Digest getIngressMac() {
+        public KeccakDigest getIngressMac() {
             return ingressMac;
         }
 
-        public SHA3Digest getEgressMac() {
+        public KeccakDigest getEgressMac() {
             return egressMac;
         }
     }
```

Comment:
Hi @nicksavers, thanks for this PR!
Did I get it right, that the latest SC version adjusted SHA-3 to the NIST standard and it is no more Keccak-256 (used in Ethereum) compatible? 
(https://medium.com/@ConsenSys/are-you-really-using-sha-3-or-old-code-c5df31ad2b0#.mnzw52i6b)